test/Constraints/function.swift

```diff
@@ -75,34 +75,40 @@ func r22451001() -> AnyObject {}
 print(r22451001(5))  // expected-error {{argument passed to call that takes no arguments}}
 
 
-// SR-590 Passing two parameters to a function that takes one argument of type Any crashes the compiler
-// SR-1028: Segmentation Fault: 11 when superclass init takes parameter of type 'Any'
```

Comment:
These two are duplicates that were not linked together on JIRA. The latter is a byproduct of the former—the one these tests are targeting—, so I removed the mention of SR-1028 and [marked it as a duplicate](https://github.com/apple/swift/issues/43640#issuecomment-1218916913) instead.

Reply:
Sounds good!